sdk/appconfiguration/Azure.Data.AppConfiguration/CHANGELOG.md

```diff
@@ -1,31 +1,34 @@
 # Release History
 
-## 1.3.0-beta.3 (Unreleased)
+## 1.3.0-beta.3 (2023-10-09)
 
 ### Features Added
 
-### Breaking Changes
+- Added a new type, `SnapshotSelector`, to encapsulate parameters like `name`, `fields`, and `status` within the `GetSnapshots` method.
 
 ### Bugs Fixed
 
 - Fixed `GetConfigurationSettings(SettingSelector)` not setting `ContentType` and `LastModified` properties [(#38524)](https://github.com/Azure/azure-sdk-for-net/issues/38524).
-
 - `FeatureFlagConfigurationSetting`  will now allow custom attributes under the `conditions` element in the setting value.  Previously, only `client_filters` was recognized and other data would be discarded.
 
 ### Other Changes
 
+- `CompositionType` has been renamed to `SnapshotComposition`.
```

Comment:
I'd remove these.   We don't typically detail renames and other changes between beta versions.  In general, we want to keep from having too many of these types of entries in the changelog so that if we ever do have GA changes such as this, the entries stand out more clearly.